### PR TITLE
jitter computation is being added

### PR DESCRIPTION
**Pull request by arifulhaque**

### CODE REVIEW

**Comment by khers** [comment]
The commit log should call out why this is necessary.

File: src/featbin/compute-kaldi-jitter-feats.cc [hunk 1]
```diff
@@ -0,0 +1,120 @@
+// featbin/compute-kaldi-pitch-feats.cc
```
Comment (khers) [inline src/featbin/compute-kaldi-jitter-feats.cc]: Either remove the file name here or update it

File: src/featbin/compute-kaldi-jitter-feats.cc [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+
+int main(int argc, char *argv[]) {
+  try {
+    using namespace kaldi;
+    const char *usage =
+        "Apply Kaldi pitch extractor, starting from wav input.  Output is 2-dimensional\n"
+        "features consisting of (NCCF, pitch in Hz), where NCCF is between -1 and 1, and\n"
+        "higher for voiced frames.  You will typically pipe this into\n"
+        "process-kaldi-pitch-feats.\n"
+        "Usage: compute-kaldi-pitch-feats [options...] <wav-rspecifier> <feats-wspecifier>\n"
```
Comment (khers) [inline src/featbin/compute-kaldi-jitter-feats.cc]: Update usage statements for the binary name (at least) also for what this does (as I assume it is different than the original program).

**Comment by arifulhaque** [comment]
This pull request is temporary and will not be merged with the master. It was only meant for Forsad to help me debug something. Sorry for any inconvenience.

**Comment by shkyang** [comment]
Looks like there were a couple of stray comments, but otherwise it looks good.

**Pull request closed without merging**